scripts/test

```diff
@@ -15,15 +15,12 @@ get_es_stats() {
 }
 
 validate_json() {
-  echo "Validating gp-data-merged.json"
-  jq '. | empty' tmp/data/input/gp-data-merged.json
-  echo "Validation exit code: $?"
-  exit_status=$(($? + exit_status))
-
-  echo "Validating mapping.json"
-  jq '. | empty' tmp/data/mapping/mapping.json
-  echo "Validation exit code: $?"
-  exit_status=$(($? + exit_status))
+  FILE=$1
```

Comment:
Is it worth using the `FILE` variable within the rest of the script rather than mixing the use of `$1` and `$FILE`. Obviously, just using `$1` removes the need for this line but I assume it has been added to make it clear the first param should be a file?